applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -114,6 +121,7 @@ Bluetooth Peripheral disconnection
 
 On a peripheral disconnection, nRF Desktop central informs the host that all the pressed keys reported by the peripheral are released.
 This is done to make sure that user will not observe a problem with a key stuck on peripheral disconnection.
+This feature is not yet implemented for boot protocole reports.
```

Comment:
This is not true. Actually FW would send empty reports in HID report protocol no matter which protocol is selected.
I think we need https://github.com/nrfconnect/sdk-nrf/pull/5584 before the release (otherwise we must add it to known issues).